code/game/gamemodes/cult/ritual.dm

```diff
@@ -218,7 +218,7 @@ This file contains the arcane tome files as well as innate cultist emergency com
 	var/rune_to_scribe
 	var/entered_rune_name
 	var/list/possible_runes = list()
-	for(var/T in subtypesof(/obj/effect/rune) - /obj/effect/rune/malformed)
+	for(var/T in subtypesof(/obj/effect/rune) - /obj/effect/rune/malformed - /obj/effect/rune)
```

Comment:
Yea that's something that snuck in when I was splitting my time between the offline/online editor when I was having it show up in my testing. Forgot to remove it here, thanks.